microsoft-edge/devtools-guide-chromium/accessibility/reference.md

```diff
@@ -5,11 +5,22 @@ author: MSEdgeTeam
 ms.author: msedgedevrel
 ms.topic: conceptual
 ms.prod: microsoft-edge
-ms.date: 06/07/2021
+ms.date: 12/01/2023
 ---
 # Accessibility-testing features
 
-To test your webpages for accessibility, first make a checklist of accessibility aspects to test, and then use the relevant DevTools features to check each aspect.
+This article lists the accessibility aspects to test, and the corresponding features in Microsoft DevTools.
```

Comment:
```suggestion
This article lists accessibility aspects to test, and the corresponding features in Microsoft DevTools.
```
softening.  scenario: manager assigns web developer a task: "Do accessibility testing, for example this checklist."

officially, the checklist of what accessibility aspects to test comes from that manager, not from these docs

Reply:
This article lists the typical accessibility aspects to test, and the corresponding features in Microsoft DevTools.